test/functorch/test_control_flow.py

```diff
@@ -256,6 +256,23 @@ def false_fn(x):
         pred = torch.tensor(False, device="cuda")
         result = cond(pred, true_fn, false_fn, [x])
         self.assertEqual(result, torch.cos(x))
+        
+    def test_cond_autograd_simple(self):
```

Comment:
Can we also add opinfo tests for cond here https://github.com/pytorch/pytorch/blob/main/torch/testing/_internal/hop_db.py? Following the map autograd tests.